Vagrantfile

```diff
@@ -0,0 +1,131 @@
+# -*- mode: ruby -*-
```

Comment:
When you have a moment, please globally adopt some boilerplate pointing all these cut and pasted Vagrantfiles to some central description of what they do and what we plan to do with them